proto/greenfield/storage/types.proto

```diff
@@ -104,6 +104,8 @@ message GroupInfo {
     (gogoproto.customtype) = "Uint",
     (gogoproto.nullable) = false
   ];
+  // extra is used to store extra info for the group
```

Comment:
We need to support update the extra field of a group.

Reply:
added